batcher/aligned-sdk/src/core/types.rs

```diff
@@ -278,3 +329,15 @@ pub enum Chain {
     Holesky,
     HoleskyStage,
 }
+
+fn get_payment_service_addr(chain_id: U256) -> Option<H160> {
+    match chain_id.as_u64() {
+        ANVIL_CHAIN_ID => H160::from_str("0x7969c5eD335650692Bc04293B07F5BF2e7A673C0").ok(),
+        HOLESKY_CHAIN_ID => H160::from_str("0x815aeCA64a974297942D2Bbf034ABEe22a38A003").ok(),
+        MAINNET_CHAIN_ID => {
+            //FIXME: Add the payment service address for mainnet
+            None
+        }
+        _ => None,
+    }
+}
```

Comment:
There is another problem with this, we are missing holesky stage

Reply:
I added a comment in the PR description regarding this. Since we need this for the audits we can do a temporary fix of manually changing the address to the one in stage. Later we can maybe change the NonceVerificationData struct.